src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -1,15 +1,37 @@
 package core.mate.academy.model;
 
-/**
- * Add some custom fields that could be only in Bulldozer
- * Do not remove no-args constructor
- */
 public class Bulldozer extends Machine {
+    private int weight;
+    private String fuel;
+
     public Bulldozer() {
     }
 
+    public Bulldozer(String name, String color, int weight, String fuel) {
+        setName(name);
+        setColor(color);
```

Comment:
Create two-args constructor in Machine class and use it here. Remember DRY principle.